setup.py

```diff
@@ -131,7 +131,11 @@ def bazel_build(self, ext: BazelExtension) -> None:  # noqa: C901
         pkgname = "google_benchmark"
         pythonroot = Path("bindings") / "python" / "google_benchmark"
         srcdir = temp_path / "bazel-bin" / pythonroot
-        libdir = Path(self.build_lib) / pkgname
+        if not self.inplace:
+            libdir = Path(self.build_lib) / pkgname
+        else:
+            build_py = self.get_finalized_command("build_py")
+            libdir = build_py.get_package_dir(pkgname)
```

Comment:
Have you tested with `self.inplace`? This looks wrong, since `get_package_dir` returns a `str`, which would make the path concatenation in `shutil.copyfile` below fail, see https://github.com/pypa/setuptools/blob/main/setuptools/command/build_py.py#L275.

At minimum, I think this should be a `Path(build_py.get_package_dir(pkgname))`.

Reply:
Yes, I tested it and it works `fp` in `libdir / fp` is `Path`, and 'str / Path' results in `Path`